### PR TITLE
Update `CONTRIBUTING.md` with Devicon line logos

### PR DESCRIPTION
**Pull request by Snailedlt**
updated CONTRIBUTING.md to show devicon line logos instead of apache line logos.

ref: https://www.script-tutorials.com/line-icons/#:~:text=Line%20icons%20is%20a%20shape%20vector%20of%20a%20symbolic%20glyph%20outline%20only%20which%20basically%20consist%20of%20fonts%20styled%20in%20css%20%7Bresize%2C%20colors%20and%20moved%7D

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ First of all, thanks for taking the time to contribute! This project can only gr
           <h4>devicon-plain.svg</h4>
       </td>
       <td style='width: 33%'>
-          <p><b>line</b>: a one-color, line version of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. </p>      
-          <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/apache/apache-line.svg' height='100px' width='100px' />
+          <p><b>line</b>: a one-color, <a href="https://www.script-tutorials.com/line-icons/#:~:text=Line%20icons%20is%20a%20shape%20vector%20of%20a%20symbolic%20glyph%20outline%20only%20which%20basically%20consist%20of%20fonts%20styled%20in%20css%20%7Bresize%2C%20colors%20and%20moved%7D">line version</a> of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. </p>      
+          <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-line.svg' height='100px' width='100px' />
           <h4>apache-line.svg</h4>
```
Comment (Panquesito7) [inline CONTRIBUTING.md:110]:
```suggestion
          <h4>devicon-line.svg</h4>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ First of all, thanks for taking the time to contribute! This project can only gr
       </td>
       <td style='width: 33%'>
           <p><b>line-wordmark</b>: similar to the above but must contain the name of the technology. Note that the icon version will be stripped of all colors so you don't have to strip beforehand.</p>      
-          <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/apache/apache-line-wordmark.svg' height='100px' width='100px' />
+          <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-line-wordmark.svg' height='100px' width='100px' />
           <h4>apache-line-wordmark.svg</h4>
```
Comment (Panquesito7) [inline CONTRIBUTING.md:127]:
```suggestion
          <h4>devicon-line-wordmark.svg</h4>
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great work! 🚀

**Comment by Snailedlt** [comment]
> Great work! 🚀

Thanks @Panquesito7 😄 
I accepted your commits too :P

**Review by amacado — CHANGES_REQUESTED** [review]
I think the href to https://www.script-tutorials.com/line-icons/ is not necessary. Would you please remove it? The updated icon version is great!

**Comment by Snailedlt** [comment]
> I think the href to https://www.script-tutorials.com/line-icons/ is not necessary. Would you please remove it? The updated icon version is great!

Done!

**Review by Panquesito7 — DISMISSED** [review]

**Comment by amacado** [comment]
> > I think the href to https://www.script-tutorials.com/line-icons/ is not necessary. Would you please remove it? The updated icon version is great!
> 
> Done!

Nope, still there.. I'm unable to approve this :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -105,9 +105,9 @@ First of all, thanks for taking the time to contribute! This project can only gr
           <h4>devicon-plain.svg</h4>
       </td>
       <td style='width: 33%'>
-          <p><b>line</b>: a one-color, line version of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. </p>      
-          <img src='https://raw.githubusercontent.com/devicons/devicon/master/icons/apache/apache-line.svg' height='100px' width='100px' />
-          <h4>apache-line.svg</h4>
+          <p><b>line</b>: a one-color, <a href="https://www.script-tutorials.com/line-icons/#:~:text=Line%20icons%20is%20a%20shape%20vector%20of%20a%20symbolic%20glyph%20outline%20only%20which%20basically%20consist%20of%20fonts%20styled%20in%20css%20%7Bresize%2C%20colors%20and%20moved%7D">line version</a> of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. </p>
```
Comment (Snailedlt) [inline CONTRIBUTING.md:108]:
```suggestion
          <p><b>line</b>: a one-color, line version of the original logo. Note that the icon version will be stripped of all colors so you don't have to strip beforehand. </p>      
```

**Comment by Snailedlt** [comment]
> > > I think the href to https://www.script-tutorials.com/line-icons/ is not necessary. Would you please remove it? The updated icon version is great!
> > 
> > 
> > Done!
> 
> Nope, still there.. I'm unable to approve this :)

You're right! Odd, I swear I removed it.

Oh well, I managed to remove it now at least 😄

**Review by Panquesito7 — APPROVED** [review]

**Review by Thomas-Boi — APPROVED** [review]
This looks good to me 👍. Thank you for working on it, it was on my TODO list.

**Pull request merged**